docs/Home.md

```diff
@@ -0,0 +1,87 @@
+# Google Kubernetes Engine Plugin Wiki
+
+The [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) (GKE) Plugin allows you to publish deployments built within Jenkins to your Kubernetes clusters running within GKE.
+
+_Plugin Information_
+
+View Google Kubernetes Engine [on the plugin site](https://plugins.jenkins.io/google-kubernetes-engine) for more information.
+
+## Usage
+
+### Enable Required APIs
+
+1. Export project.
+    ```bash
+    export PROJECT=$(gcloud info --format='value(config.project)')
+    ```
+2. Enable GKE, Google Compute Engine (GCE), Service Management, and Cloud Resource Manager APIs.
+    ```bash
+    gcloud services enable --project $PROJECT compute.googleapis.com container.googleapis.com servicemanagement.googleapis.com cloudresourcemanager.googleapis.com
+    ```
+
+### IAM Credentials
+
+1. Create a service account using the Google Cloud SDK.
+    ```bash
+    gcloud iam service-accounts create jenkins-gke
+    ```
+1. Add the KubernetesEngineAdmin, serviceAccountUser, and computeNetworkViewer roles to the service account:
```

Comment:
Suggest changing to "Add required service account roles", since you enumerate them below already.